client/.storybook/preview.js

```diff
@@ -1,3 +1,15 @@
+import React from "react";
+import GlobalStyles from "../src/GlobalStyles";
 export const parameters = {
   actions: { argTypesRegex: "^on[A-Z].*" },
 };
+const withGlobalStyles = (Story, context) => {
```

Comment:
Instead of using this higher order function `withGlobalstyles`, which is not as modern as hooks and not up to your standards (in terms of code style), I would recommend using a decorator instead. Basically what you want to do is wrap all stories with `GlobalStyles`
https://storybook.js.org/docs/react/writing-stories/decorators

For an example implementation check out the `environmentDecorator.js` and `preview.js` in this repository: https://github.com/Caruki/show-cased/tree/master/client/.storybook

Don't forget to leave a ⭐ if it helped 🐜

Reply:
Had to correct that comment :)